frontend/ui/elements/filemanager_menu_order.lua

```diff
@@ -128,6 +128,7 @@ local order = {
     },
     main = {
         "history",
+        "collections",
         "open_last_document",
         "----------------------------",
```

Comment:
For both reader and filebrowser hamburger menu, I'd rather see `history` and `open_last/previous_document` grouped together. As collections/favorites is not related to history/recent file, may be:
```
        "history",
        "open_last_document",
        "----------------------------",
        "collections",
        "----------------------------",
```